Mathlib/CategoryTheory/Filtered/Flat.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2025 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer
+-/
+import Mathlib.CategoryTheory.Filtered.CostructuredArrow
+import Mathlib.CategoryTheory.Filtered.Connected
```

Comment:
```suggestion
```

Reply:
I wonder why the shake linter didn't catch this :thinking: